Helpers/QuoteHelper.php

```diff
@@ -52,6 +54,11 @@ class QuoteHelper
      */
     private $quoteRepository;
 
+    /**
+     * @var null
+     */
+    private $quoteId;
```

Comment:
You are replacing `$quote` by `$quoteId` 😛

Reply:
I have remote quote from eligibility  - I have save quoteId in quoteHelper